src/main/java/net/neoforged/neoform/runtime/cli/RunNeoFormCommand.java

```diff
@@ -270,28 +280,42 @@ private static void applyNeoForgeProcessTransforms(NeoFormEngine engine, JarFile
         graph.setResult(ResultIds.GAME_JAR_WITH_NEOFORGE, compiledWithNeoForgeOutput);
         graph.setResult(ResultIds.GAME_JAR_WITH_SOURCES_AND_NEOFORGE, sourcesAndCompiledWithNeoForgeOutput);
 
-        // Support for binary patches
-        var renamedOutput = graph.getRequiredOutput("rename", "output");
+        applyNeoForgeBinaryPatchProcessTransforms(engine, neoforgeZipFile, neoforgeConfig, neoforgeClassesZip);
+
+    }
+
+    private static void applyNeoForgeBinaryPatchProcessTransforms(NeoFormEngine engine,
+                                                                  JarFile neoforgeZipFile,
+                                                                  NeoForgeConfig neoforgeConfig,
+                                                                  ZipFile neoforgeClassesZip) {
+        var graph = engine.getGraph();
+        var patchBaseJar = graph.getResult(ResultIds.VANILLA_DEOBFUSCATED);
+
         engine.addDataSource("patch", neoforgeZipFile, neoforgeConfig.binaryPatchesFile());
-        var binaryPatchOnlyOutput = createBinaryPatch(graph, renamedOutput, neoforgeConfig.binaryPatcherConfig());
-        var binaryPatchOutput = createCopyUnpatchedClasses(graph, renamedOutput, binaryPatchOnlyOutput);
+        var binaryPatchOutput = createBinaryPatch(graph, patchBaseJar, neoforgeConfig.binaryPatcherConfig());
+        binaryPatchOutput = createCopyUnpatchedClasses(graph, patchBaseJar, binaryPatchOutput);
+
+        // For binpatches we also need to consider Access Transforms / Interface Injection
+        // However, we have to force-create the node here, since it's placement with NeoForge enabled
+        // is very different from when it is placed by the NeoForm process.
+        binaryPatchOutput = createBinaryDevTransformNode(graph, binaryPatchOutput.asInput());
```

Comment:
Didn't we forget to `.setAccessTransformersData(List.of("neoForgeAccessTransformers"))`?